cmake/modules/AddSwift.cmake

```diff
@@ -1034,7 +1055,7 @@ function(_add_swift_library_single target name)
     set_target_properties("${target}"
       PROPERTIES
       INSTALL_NAME_DIR "${install_name_dir}")
-  elseif("${SWIFTLIB_SINGLE_SDK}" STREQUAL "LINUX" AND NOT "${SWIFTLIB_SINGLE_SDK}" STREQUAL "ANDROID")
+  elseif("${SWIFTLIB_SINGLE_SDK}" STREQUAL "LINUX")
```

Comment:
Hmm, yeah, it might be nice to introduce a `/usr/lib/swift/android`.

Reply:
Yeah, I'll probably need to add that when I try building the Termux package. For now, this change is simply logical: the SDK string can't be ANDROID when it's already set to LINUX.

Reply:
Added the install RPATH below also.